modelservice/src/test/java/com/expedia/adaptivealerting/modelservice/util/ErrorDetailsTest.java

```diff
@@ -1,3 +1,18 @@
+/*
+ * Copyright 2018-2019 Expedia Group, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package com.expedia.adaptivealerting.modelservice.util;
 
 public class ErrorDetailsTest {
```

Comment:
@bibinss Can I remove this file? Since it's empty and of no use.

Reply:
yes please

Reply:
Sure